scripts/core/registry.yml

```diff
@@ -235,6 +235,9 @@ etors:
 - name: PLATFORM_CREATE_WITH_NATIVE_HANDLE
   desc: Enumerator for $xPlatformCreateWithNativeHandle
   value: '76'
+- name: PLATFORM_GET_BACKEND_OPTION
+  desc: Enumerator for $xPlatformGetBackendOption
+  value: '114'
```

Comment:
could we add this at the end? this might produce confusion for future changes, where a developer might think last enum is the last in the list, as typical.

Reply:
Good spot! This is an autogenerated file, but i've modified the script so that it will sort the keys in ascending order.

Reply:
see [here](https://github.com/oneapi-src/unified-runtime/pull/436/files#diff-9fd58b4368cd26364ecae76972b060dcf36a6546c8addaefbc2cb2edea428c18R55)